### PR TITLE
common-foo: Move DRCT.retreat to DRC to fix circular dependency probl…

### PR DESCRIPTION
**Pull request by Sarvatt**
…em when auto stating afk.lic after equipmanager was added to its custom_require

### CODE REVIEW

**Comment by Sarvatt** [comment]
both of these scripts get loaded at login so it should be ok to have a flag day moving it like this without a transition

**Comment by Sarvatt** [comment]
hmm, it may be worth holding off on the equipmanager change at least until the common files propagate, since a lot of scripts do EquipManager.new? Or am I misunderstanding.

File: common-travel.lic [hunk 1]
```diff
@@ -75,10 +75,7 @@ module DRCT
   def retreat(ignored_npcs = [])
     return if (DRRoom.npcs - ignored_npcs).empty?
```
Comment (rpherbig) [inline common-travel.lic]: ❓ Why wasn't this moved?

**Comment by rpherbig** [comment]
❓ What's the circular dependency here?

**Comment by rcuhljr** [comment]
common-arcana requires common-travel and common-travel requires common-arcana. SHAME ON YOU SHELT.

**Comment by rpherbig** [comment]
> hmm, it may be worth holding off on the equipmanager change at least until the common files propagate, since a lot of scripts do EquipManager.new? Or am I misunderstanding.

Yeah, I think what we want to do is add `retreat` to `DRC`, have the `DRCT` version do nothing but call the new one, and point `DRCA` at the new one (removing the dependency on `DRCT` in the process).

Then a second PR updates scripts to point to `DRC` and removes the `DRCT` function.

File: common-travel.lic (hunk 1 above)
Comment (Sarvatt) [reply common-travel.lic]: do we want to add drinfomon to custom_require in common.lic?

Comment (rcuhljr) [reply common-travel.lic]: Probably the best choice.

Comment (Sarvatt) [reply common-travel.lic]: I'm an idiot, was thinking adding drinfomon to common would add another circular dependency but drinfomon doesn't include common, and common already includes drinfomon. fixed.

File: common-travel.lic [hunk 2]
```diff
@@ -75,10 +75,7 @@ module DRCT
...
 
-    escape_messages = ['You are already as far away as you can get', 'You retreat from combat', 'You sneak back out of combat']
-    until escape_messages.include?(DRC.bput('retreat', *(escape_messages + ['retreat', 'You try to back ', 'sneak', 'grip remains solid', 'You must stand first', 'grip on you', 'You stop advancing'])))
-      DRC.fix_standing
-    end
+    DRC.retreat
```
Comment (Sarvatt) [inline common-travel.lic]: should this be `retreat(ignored_npcs = [])`?

Comment (rcuhljr) [reply common-travel.lic]: it should be `DRC.retreat(ignored_npcs)`

File: common-travel.lic (hunk 1 above)
Comment (rpherbig) [inline common-travel.lic]: 💡 You can delete this guard clause since it's duplicated in `DRC`

**Pull request merged**